CHANGELOG.rst

```diff
@@ -3,6 +3,11 @@ Changelog
 =========
 
 
+1.1.0 (2019-08-21)
+------------------
+
+* refactoring
```

Comment:
Judging by the diff I think there is something more than just `refactoring` going on ;)
I suggest to elaborate a bit more about the changes

Reply:
I added more informations about changes.